apps/sms/services/js/messaging/messaging_service.js

```diff
@@ -12,7 +13,7 @@ if (!('BridgeServiceMixin' in self)) {
 (function(exports) {
 
 const SERVICE_NAME = 'messaging-service';
-const MOBILE_MESSAGE_CLIENT_NAME = 'mozMobileMessageShim';
+const MOBILE_MESSAGE_CLIENT_NAME = 'moz-mobile-message-shim';
```

Comment:
we don't have that many objects and we control the names so I'm not too concerned here.